src/types/bytes.rs

```diff
@@ -78,6 +78,7 @@ impl PyBytes {
     /// leading pointer of a slice of length `len`. [As with
     /// `std::slice::from_raw_parts`, this is
     /// unsafe](https://doc.rust-lang.org/std/slice/fn.from_raw_parts.html#safety).
+    /// If `ptr` is null, the contents of the bytes object are uninitialized.
```

Comment:
Hmm. Bytes objects aren't really supposed to be mutated, so having uninitialised data here seems not ideal. Should we deprecate this and go for a slice here too?